tests/system/robot/runLocalCaptioner.robot

```diff
@@ -0,0 +1,28 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2025 NV Access Limited, tianze
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+*** Settings ***
+Documentation	Local captioner tests
+Force Tags	NVDA	smoke test	startupShutdown
```

Comment:
Can you renamed this file to automatedImageDescriptions.robot

You'll also need to add the `imageDescriptions` tag to `.github\workflows\testAndPublish.yml` `systemTests.needs.matrix`

```suggestion
Force Tags	NVDA	smoke test	imageDescriptions
```